pkg/apis/core/validation/validation.go

```diff
@@ -2286,6 +2289,41 @@ func validateDNSPolicy(dnsPolicy *core.DNSPolicy, fldPath *field.Path) field.Err
 	return allErrors
 }
 
+const (
+	maxDNSNameservers     = 3
```

Comment:
Comment that this is because of long-standing limits in Linux.

Reply:
Added comments.